fop/fom.c

```diff
@@ -302,12 +302,6 @@ static bool fom_is_blocked(const struct m0_fom *fom)
 		M0_IN(fom->fo_thread->lt_state, (BLOCKED, UNBLOCKING));
 }
 
-/* Returns fom from state machine m0_fom::fo_sm_state */
-static inline struct m0_fom *sm2fom(struct m0_sm *sm)
```

Comment:
Can we add "M0_UNUSED" here and not to remove this function?
Just like other places?

Reply:
Sure, will do this. Would you like me to keep other removed functions or only this one? I also removed:
- rm/rm.c::credit_to_resource_type()
- rm/rm.c::rem_incoming_to_resource_type()
- motr/st/st.c::skipspaces()

Reply:
Yes. I'd like them to be kept. They might be used later.

Reply:
Done.